Data Structures/Stack/Python/stack.py

```diff
@@ -0,0 +1,41 @@
+'''
+ * This class Implements Stack
+ * @author asterisk30
+ */
+'''
+class Stack:
+    def __init__(self):
+        self.items = [] 
+ 
+    def is_empty(self):
+        return self.items == [] 
+    def push(self, data):
+        self.items.append(data) 
+ 
+    def pop(self):
+        return self.items.pop() 
+ 
+    def disp(self):
+            print(self.items,end=" ")
+ 
+#driver program
+if __name__=='__main__':
+    s = Stack()
+    while True:
```

Comment:
You can remove this loop.
You just need to demonstrate the stack so a simple program 
```
stack=Stack()
print(stack.is_empty())
stack.push(3)
print(stack.is_empty())
print(stack.pop(3))
```
This is clearer